docs/source/ops/connect/helm.rst

```diff
@@ -0,0 +1,281 @@
+`Kafka <https://kafka.apache.org/>`_, etc.
+
+You can find external documentation on, how to setup `ElasticSearch` with
+`Filebeat` and `Kibana` for analyzing logs on your Kubernetes cluster
+`here <https://www.deepnetwork.com/blog/2020/01/27/ELK-stack-filebeat-k8s-deployment.html>`_.
+
+As of 1.18.0, the :ref:`HTTP JSON API <json-api>` component in the Helm chart produces
+JSON-encoded logs. Other components log in somewhat less structured formats.
+
+Daml Metrics Options
+--------------------
+
+The Daml Driver for PostgreSQL instance and the HTTP JSON API instances started
+by the Helm chart are configured to expose Prometheus metrics on a port named
+`metrics`, using the appropriate annotations. This means that, if you are
+running a cluster-wide Prometheus instance, the relevant metrics should be
+collected automatically.
+
+See each component's documentation for details on the metrics exposed:
+
+- `Daml Driver for PostgreSQL </daml-driver-for-postgresql/#types-of-metrics>`_
+- :ref:`JSON API metrics <json-api-metrics>`
+
+Upgrading
+---------
+
+.. note::
+
+   This section only makes sense with the ``production`` flag set to ``true``.
+
+Upgrading the Daml Connect version should be done by uninstalling the exiting
+Helm chart, waiting for all of the pods to stop, and then installing a higher
+version. Destroying all of the components is a safe operation because all of
+the state is stored in the provided database coordinates. There is no
+additional state within the components themselves.
+
+The components are not designed for running concurrently with older versions,
+so it is imperative to wait for the existing Helm chart components to be
+completely shut down before installing the new one. Do not try to upgrade in
+place.
+
+Assuming you do not change the database coordinates, you should have data
+continuity through the upgrade.
+
+Backing Up
+----------
+
+.. note::
+
+   This section only makes sense with the ``production`` flag set to ``true``.
+
+For a production setup, you should be providing the Helm chart with external
+database coordinates. The simplest approach here is to periodically back up
+those databases as a whole, just like you would any other database.
+
+If you want to be more fine-grained, you *may* decide to **not** backup the
+database used by the HTTP JSON API Service instances. Note that it is
+imperative that you still backup the databases for the other components (Trigger
+Service and Daml Driver for PostgreSQL) if you are running them.
+
+If you are running the Helm chart solely for the HTTP JSON API Service
+(connected to an external Ledger API server), then you can eschew backing up
+entirely, as the database for the HTTP JSON API Service is an
+easy-to-reconstruct cache. This assume that, in this setup, the data store of
+the Ledger API server is, itself, properly backed up.
+
+Securing Daml Connect
+---------------------
+
+The Helm chart assumes that the Kubernetes environment itself is trusted, and
```

Comment:
Speaking about capitalization, check whether "chart" should be capitalized. To me a "chart" is some visual representation of data and "Chart" is the given name of Helm bundles (I'm quite curious of why they chose that name though).

Reply:
Looking through the helm.sh site, I can only find one instance of "Helm Chart" outside a title, and lots of instances of "Helm chart" through the text. I do think it looks a bit weird too, but :shrug:.